README.md

```diff
@@ -28,6 +28,163 @@ Plug 'tribela/vim-transparent'
 [Vim-Plug]: https://github.com/junegunn/vim-plug
 
 
+## Configuration
+
+### Minimal configuration
+
+```vim
+Plug 'tribela/vim-transparent'
+
+" Clear the background of highlight groups in g:transparent_groups.
+let g:transparent_groups = ['Normal', 'Comment', 'Constant', 'Special', 'Identifier',
```

Comment:
As default values are implemented, This config is not needed on minimal configuration